docs/devguide.md

```diff
@@ -151,6 +151,8 @@ To build the service-catalog client, `svcat`:
 
     $ make svcat
     
+The svcat cli binary is located at `bin/svcat/svcat`.
+    
```

Comment:
Is there a historical reason that output is in a subdirectory?

Reply:
When we do a release, we build binaries for every platform so the tree looks like this:

```
- bin
  - svcat
    - VERSION
      - OS
        - ARCH
           - svcat(.exe)
```
In order to avoid coliding with that, we output to bin/svcat/svcat. Suggestions welcome on making this more simple. :-D

Reply:
Are they all built at the same time?

Reply:
I may be the only one who calls that locally, normally the cross build is only done during CI.

Are you suggesting telling me to man up and write to bin/svcat for `make svcat`? 😀 I wouldn't be against that, just to make life easier for everyone. What do you think?